entries/h/humana.com.json

```
@@ -0,0 +1,13 @@
+{
+    "Humana": {
+      "domain": "humana.com",
```

`https://humana.com/` doesn't respond to requests.

```suggestion
      "domain": "humana.com",
      "url": "https://www.humana.com/",
```